cmd/agent/app/check.go

```diff
@@ -44,6 +45,7 @@ func init() {
 	checkCmd.Flags().IntVar(&checkPause, "pause", 0, "pause between multiple runs of the check, in milliseconds")
 	checkCmd.Flags().StringVarP(&logLevel, "log-level", "l", "", "set the log level (default 'off')")
 	checkCmd.Flags().IntVarP(&checkDelay, "delay", "d", 100, "delay between running the check and grabbing the metrics in miliseconds")
+	checkCmd.Flags().BoolVarP(&formatJSON, "json", "", false, "format aggregator output as json")
```

Comment:
Can we update the help text a bit to reflect the fact that you don't get all the same data in json, only the series data? Or, better, could we serialize the Check status section to json as well?